### PR TITLE
Sundhar dev

### PR DESCRIPTION
**Pull request by sundharm**
[AB#126](https://dev.azure.com/Tringo/cabcbfb6-3a22-44cf-ac98-b3810925a870/_workitems/edit/126)

### CODE REVIEW

File: app/src/components/markers/priceTagMarker.tsx [hunk 1]
```diff
@@ -63,13 +65,21 @@ export class PriceTagMarker extends Component<MarkerProps, flightSearchParameter
...
         this.addListener();
     }
 
     componentDidUpdate(): void {
         this.addListener();
     }
 
+    onHover(): void {
+        let tag = findDOMNode(this) as Node;
+        tag.addEventListener('click', () => {
```
Comment (s3419069) [inline app/src/components/markers/priceTagMarker.tsx:78]: just wonder, is it really should be 'click' eventlistener to fire hover event?? click->hover?

Comment (sundharm) [reply app/src/components/markers/priceTagMarker.tsx:78]: We are using custom markers and the lat long which we have is not precisely the same as what google maps would return on click/hover. Not sure if we can have a line on hover using custom marker, that is why I put click.

Comment (s3419069) [reply app/src/components/markers/priceTagMarker.tsx:78]: can you send a screenshot with the example? but it must show the line on hover, is that how it works?

Comment (sundharm) [reply app/src/components/markers/priceTagMarker.tsx:78]: maybe it is possible, but we need to handle events on the map, I am not sure how to handle a hover event on the marker and correspond it to a hover event on the map. U need a screenshot of what we have right now?

Comment (s3419069) [reply app/src/components/markers/priceTagMarker.tsx:78]: 'lat long which we have is not precisely the same as what google maps would return on click/hover'.
If it show a line on Click, but not on hover, we won't merge it with dev, sorry. 
what if you change it to tag.addEventListener('**hover**', ...)

**Comment by s3419069** [comment]
so much work to redo. I will open other PR

**Pull request closed without merging**